<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QWidget" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Data Scheme</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioButtonMemory">
             <property name="text">
              <string>Memory</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/Icons/ram.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonDB">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>DB</string>
             </property>
             <property name="icon">
              <iconset theme="applications-science">
               <normalon>:/Icons/database.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetDB">
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <row>
           <property name="text">
            <string>DB Type</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </row>
          <row>
           <property name="text">
            <string>DB Name</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </row>
          <row>
           <property name="text">
            <string>DB Path</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </row>
          <column>
           <property name="text">
            <string>INFO</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>NONE</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>NONE</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>NONE</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
